app.js

```diff
@@ -38,7 +38,7 @@ if (app.get("env") === "development") {
     res.status(err.status || 500);
     res.render("error", {
       message: err.message,
-      error: err
+      error: err,
```

Comment:
Why do you updated code styles?

Reply:
`npm run test` reported those as failures. Happy to revert!

Reply:
Can be your revert these changes. I already spend all my free time for preparing PostCSS 8 release?

Reply:
Reverted 😀